packages/client/src/data_commons_client.test.ts

```diff
@@ -422,14 +434,28 @@ describe("DataCommonsWebClient", () => {
 
   test("Get data rows", async () => {
     const response = await client.getDataRows({
+      childType: "State",
+      parentEntity: "country/MOCK",
+      variables: ["Has_Data", "No_Data"],
+    });
+    response.forEach((row) => {
+      expect(
+        ["Has_Data", "No_Data"].indexOf(row.variable.dcid)
+      ).toBeGreaterThanOrEqual(0);
+    });
+    expect(Object.keys(response).length).toBe(3);
+  });
+
+  test("Get data rows", async () => {
```

Comment:
Should this be named "Get data rows grouped by entity" instead?

Reply:
Good catch-updated